engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/date-type-errata-filter.controller.js

```diff
@@ -35,6 +35,10 @@ angular.module('Bastion.content-views').controller('DateTypeErrataFilterControll
                     $scope.types[type] = true;
                 }
             });
+
+            if (!angular.isDefined($scope.rule['date_type'])) {
```

Comment:
Could you use https://docs.angularjs.org/api/ng/function/angular.isUndefined instead of the not?